Project.swift

```diff
@@ -552,6 +552,14 @@ func targets() -> [Target] {
 
 let project = Project(
     name: "Tuist",
+    options: [
+        .textSettings(
+            .init(
+                indentWidth: 4,
+                tabWidth: 4
+            )
+        )
+    ],
```

Comment:
nitpick: maybe I would put at the end of the signature as i think other parameters are more important (e.g. settings and targets)

Reply:
I thought this way, we start with project settings like `organizationName`, `options` and etc. and then continue with project structure `targets`, `schemes` and else. 

But not sure what will be better.